src/components/CollapserRow.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+import Collapser from './Collapser';
+import { Icon } from '@newrelic/gatsby-theme-newrelic';
+
+const collapserIcon = (isOpen) => css`
+  margin-left: auto;
+  transition: transform 0.6s ease;
+  color: var(--accent-text-color);
+  ${isOpen &&
+  `
+transform: rotate(180deg);`}
+`;
+
+const CollapserRow = ({ title, id, openByDefault, children }) => {
+  const [isOpen, toggleOpen] = useState(openByDefault);
+  const [setHeight, setHeightState] = useState(isOpen ? 'auto' : '0px');
+
+  const content = useRef(null);
+  const toggleCollapser = () => {
+    toggleOpen(!isOpen);
+    setHeightState(isOpen ? '0px' : `${content.current.scrollHeight}px`);
+  };
+
+  return (
+    <Collapser>
+      <div
+        css={css`
+          display: flex;
+          flex-direction: column;
+          border-radius: 3px;
+          margin-bottom: 8px;
+        `}
+      >
+        <button
+          onClick={toggleCollapser}
+          type="button"
+          css={css`
+            cursor: pointer;
+            padding: 0.75rem;
+            border: 1px solid var(--border-color);
+            background-color: inherit;
+            border-radius: 3px;
+            display: flex;
+            align-items: center;
+            transition: background-color 0.6s ease;
+            &:hover,
+            &:focus {
+              background-color: var(--color-neutrals-100);
+              outline: none;
+              .dark-mode & {
+                background-color: var(--color-dark-100);
+              }
+            }
+          `}
+        >
+          <h5
+            id={id}
+            css={css`
+              font-family: 'Open Sans', sans-serif;
+              font-weight: 600;
+              font-size: 14px;
+              text-align: left;
+              margin-top: 0;
+              margin-bottom: 0;
+              font-size: 16px;
```

Comment:
Should we try and maintain the same defaults for an `h5` as we do the rest of the site? In that case, you'd likely want to remove the `font-size`, and `font-weight` properties. I think the `margin-top: 0` and `margin-bottom: 0` make sense here since you're trying to remove it for the purposes of this component. Do we need to override the default size and weight for the `h5`?

Reply:
when i removed the font-size i feel like it made the text just a bit too small. 16px or 1rem seemed about right. i get what you're saying though. would be more ideal if we didn't override default heading style.

we could always use another tag though too. i used `<h5>` because the current site primarily uses `<h2>` and `<h3>`, and on rare occasion `<h4>`.